osrf_testing_tools_cpp/test/memory_tools/test_memory_tools.cpp

```diff
@@ -200,9 +200,9 @@ TEST(TestMemoryTools, test_example) {
   }, "unexpected malloc");
   // There are also explicit begin/end functions if you need variables to leave the scope
   osrf_testing_tools_cpp::memory_tools::expect_no_malloc_begin();
-  int result = my_second_function(1, 2);
+  int output = my_second_function(1, 2);
   osrf_testing_tools_cpp::memory_tools::expect_no_malloc_end();
-  EXPECT_EQ(result, 3);
+  EXPECT_EQ(output, 3);
```

Comment:
I have no particular problem with this change, but why is it needed?

Reply:
I did it to avoid shadowing the `result` variable in several functions.

Reply:
Oh, I missed the shadows down below.  Makes sense, thanks.